src/shared/utils/logger.js

```diff
@@ -47,6 +47,7 @@ if (isServerSide()) {
       }),
       headers: {
         'Content-Type': 'application/json',
+        Authorization: `ApiKey ${process.env.SERVER_API_KEY}`,
```

Comment:
This is server-side code only, should be `import config from 'config';` or `import config from 'utils/config';`, if you don't wanna think whether it is server- or client-side, and then `config.SERVER_API_KEY`: reading it directly from `SERVER_API_KEY` misses the case when the key is set via config, and not via the env var.